new/LIT7439MMEAhmad.xml

```diff
@@ -0,0 +1,85 @@
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2025-05-13">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Strelcyn1976Catalogue"/>
+                        <citedRange unit="page">38</citedRange>
+                    </bibl>
+                </listBibl>
+                <listRelation>
+                    <listRelation>
+                        <relation name="saws:formsPartOf" active="LIT7439MMEAhmad"
+                            passive="LIT7420MirME"/>
+                        <relation name="saws:formsPartOf" active="LIT7439MMEAhmad"
+                            passive="LIT7419HagDosME"/>
+                        <relation name="ecrm:P129_is_about" active="LIT7439MMEAhmad" passive="PRS6445Madhanin"/>
+                        <relation name="ecrm:P129_is_about" active="LIT7439MMEAhmad" passive="PRS1522Ahmadb"/>
+                    </listRelation>
+                </listRelation>
+            </div>
+            <div type="edition">
+                <div type="textpart" subtype="incipit" xml:lang="gez">
+                    <note>This incipit is taken from <ref type="mss" corresp="ANLcr13"/>.</note>
+                    <ab>ተአምሪሁ፡ <gap reason="ellipsis"/> ስምዑ፡
+                        አበዊነ፡ ወአሐዊነ፡ እለ፡
+                        በክርስቶስ፡ ሀሎክሙ፡ ሐዲስ፡ ተአምሪሁ፡ ለአቡነ፡ መድኃኒነ፡ እግዚእ፡ <gap reason="ellipsis"/>
+                        ሀለወክሙ፡ ጽሑፈ፡ በዜና፡ ገድሉ፡ ባሕቱ፡ ንነግር፡ ዘኮነ፡ ይእዜ፡ በመዋዕሊነ፡ እንዘ፡ ንጉሥነ፡ ሠርፀ፡ ድንግል፡ ወመኰንንነ፡ ይስሐቅ፡ ወአበ፡
```

Comment:
Would it make sense to mark up these people? Or to signal somehow that they are mentioned here? (In the abstract maybe – it seems too much to have "is about" relations…)